app/views/projects/_form.html.erb

```diff
@@ -22,10 +22,12 @@
     <%= form.collection_check_boxes :user_ids, User.all, :id, :name %>
   </div>
 
-  <div class="field checkbox-list">
-    <%= form.label :samples %>
-    <%= form.collection_check_boxes :sample_ids, Sample.all, :id, :name %>
-  </div>
+  <% if false %>
```

Comment:
if false?

Reply:
yeah. it's a way to comment it out for now. the sample list is getting too huge.